### PR TITLE
Fix composer install procedure

### PR DESCRIPTION
**Pull request by nervo**
See: https://github.com/composer/composer/issues/5135

### CODE REVIEW

File: source/setup.rst [hunk 1]
```diff
@@ -100,7 +100,7 @@ The installation is possible by adding our own repository
 
 .. code-block:: sh
 
-    composer create-project phpmyadmin/phpmyadmin --repository-url=https://www.phpmyadmin.net/packages.json
+    composer create-project phpmyadmin/phpmyadmin --repository=https://www.phpmyadmin.net
```
Comment (stof) [inline source/setup.rst:103]: the option name should not be changed. It is `--repository-url`

**Comment by stof** [comment]
note that this change is necessary only because of a composer bug. For people using a nightly build, using the URL of the packages.json file should work properly (but `composer self-update` does not use nightly builds by default anymore)

File: source/setup.rst (hunk 1 above)
Comment (Seldaek) [inline source/setup.rst:103]: No it's not. That has been deprecated as --repository handles URLs _and_ complete repo definitions.

**Comment by nijel** [comment]
Given that we do not want to break for people not using latest composer, I don't think this is good to change now. 

PS: This repository is only for translating the documentation, the source is https://github.com/phpmyadmin/phpmyadmin/tree/master/doc.

**Comment by nervo** [comment]
Well, given that composer is now 1.0 stable for few days, and that this is the official way of using custom repository, i think you should re-consider :)
But that's up to you...

**Comment by nijel** [comment]
I don't think that few days are enough for most people to ugprade :-)

**Comment by Seldaek** [comment]
@nijel you should wait a while if you wanna play it safe, but IMO you should merge this eventually, as --repository-url is kinda deprecated (we don't issue a warning for it, but might at some point).

**Comment by nijel** [comment]
I think it's best to merge at point it will issue deprecation warnings.

**Pull request closed without merging**